<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopMenu</class>
 <widget class="QWidget" name="TopMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>24</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(67, 67, 67);
font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>深度看图</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_edit">
   <property name="geometry">
    <rect>
     <x>1212</x>
     <y>0</y>
     <width>68</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_create">
   <property name="geometry">
    <rect>
     <x>1144</x>
     <y>0</y>
     <width>68</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ret">
   <property name="geometry">
    <rect>
     <x>1076</x>
     <y>0</y>
     <width>68</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
